config/config.exs

```diff
@@ -15,15 +15,12 @@ use Mix.Config
 #       format: "$date $time [$level] $metadata$message\n",
 #       metadata: [:user_id]
 
-
 # It is also possible to import configuration files, relative to this
 # directory. For example, you can emulate configuration per environment
 # by uncommenting the line below and defining dev.exs, test.exs and such.
 # Configuration from the imported file will override the ones defined
 # here (which is why it is important to import them last).
 #
 
-if Mix.env == :test ||
-    Mix.env == :dev ||
-    Mix.env == :docs, do: import_config "#{Mix.env}.exs"
-
+if Mix.env() == :test || Mix.env() == :dev || Mix.env() == :docs,
```

Comment:
Maybe this should be:

```ex
if Mix.env() in [:test, :dev, :docs], do: import_config("#{Mix.env()}.exs")
```

Reply:
@scrogson we do not need to do this.

Nobody will use `prod` on this package and when you use it in another project the `config/config.exs` will be ignore from this package anyway.

Right?!